talentlink-connector/processes/rest/Attachment.p.json

```diff
@@ -0,0 +1,81 @@
+{
+  "format" : "10.0.0",
+  "id" : "1882E31A11B7E690",
+  "kind" : "CALLABLE_SUB",
+  "config" : {
+    "data" : "rest.AttachmentData"
+  },
+  "elements" : [ {
+      "id" : "f0",
+      "type" : "CallSubStart",
+      "name" : "getAttachment(Long)",
```

Comment:
I'd ad the 'connector' tag onto this start, so that users of the connector can easily spot it and select it via the 'extensions' menu in the process-editor